app/components/about-component.tsx

```diff
@@ -0,0 +1,25 @@
+interface AboutProps {
+  image?: string;
+  title?: string;
+  description?: string;
+}
+
+const AboutComponent = ({
+  image,
+  title,
+  description,
+}: AboutProps) => {
+  return (
+    <div className="flex flex-col items-center text-center">
+      <img
+        src={image}
```

Comment:
for example:
```suggestion
                src={image ?? 'https://i.imgur.com/BlBH8Ak.jpeg' }
```
random panda picture for now